mpl_interactions/controller.py

```diff
@@ -291,6 +293,16 @@ def save_animation(
             else:
                 N = int((max_ - min_) / slider.valstep)
                 step = slider.valstep
+        elif not _not_ipython and isinstance(slider, (FloatSlider, IntSlider)):
```

Comment:
I think this could also be made to work for `SelectionSliders` right?

Reply:
I guess any [widget](https://ipywidgets.readthedocs.io/en/7.6.3/examples/Widget%20List.html) that has a `min`, `max`, and `step` should work. Best would be to use some base class that has those attributes in common (maybe `ipywidgets.widgets.Widget`? idk, hard to figure out that inheritance tree). Or else a `Protocol` or using `hasattr`. But then again, those checks could be too wide...

Reply:
The inheritance tree for ipywidgets is really about what sort of values they hold, not whether they are sliders. So they inheirt from things like `BoundedInt` https://github.com/jupyter-widgets/ipywidgets/blob/e0d41f6f02324596a282bc9e4650fd7ba63c0004/ipywidgets/widgets/widget_int.py#L78 so we have to check out sliders one by one